lib/jdomodels/src/main/resources/schema/functions/GetEntityPath.ddl.sql

```diff
@@ -0,0 +1,35 @@
+CREATE FUNCTION `getEntityPath`(inputEntityId BIGINT) RETURNS varchar(1000) CHARSET utf8mb4
```

Comment:
An alternative to consider is to use a recursive cte (See https://dev.mysql.com/doc/refman/8.0/en/with.html#common-table-expressions-recursive), something on the lines of:

`WITH RECURSIVE PATH(ID, PARENT_ID, DISTANCE) AS (
        -- Initial row set, replace ? with the id of the entity
	SELECT ID, PARENT_ID, 1
	FROM JDONODE WHERE ID = ?
	
        -- Distinct allows loop detection as well
	UNION DISTINCT
	
	-- Recursion
	SELECT JDONODE.ID, JDONODE.PARENT_ID, PATH.DISTANCE+ 1
	FROM JDONODE JOIN PATH ON (JDONODE.ID = PATH.PARENT_ID)
        -- Break point
	WHERE JDONODE.ID IS NOT NULL AND DISTANCE <= 100
)
SELECT * FROM PATH ORDER BY DISTANCE;`

Reply:
Interesting.
I setup a test on a hierarchy 12 deep to compare this query to the function and they have identical run times (not surprising).  I think the main advantage of the "WITH RECURSIVE" approach is we can select additional columns like name and type.  With the function, I need a second call to gather name and type for the results.